datadog_checks_base/datadog_checks/base/utils/time.py

```diff
@@ -6,61 +6,29 @@
 from datetime import datetime
 from time import time as epoch_offset
 
-import pytz
-from six import PY2
+from dateutil.tz import UTC
 
-UTC = pytz.utc
-EPOCH = datetime(1970, 1, 1, tzinfo=UTC)
+EPOCH = datetime.fromtimestamp(0, UTC)
 
 
-if PY2:
-
-    def get_timestamp(dt=None):
-        """
-        Returns the number of seconds since the Unix epoch.
-        If `dt` is not specified or `None`, the current time in UTC is assumed.
-        """
-        if dt is None:
-            return epoch_offset()
-
-        return (dt - EPOCH).total_seconds()
-
-
-else:
-
-    def get_timestamp(dt=None):
-        """
-        Returns the number of seconds since the Unix epoch.
-        If `dt` is not specified or `None`, the current time in UTC is assumed.
-        """
-        if dt is None:
-            # NOTE: Although the epoch is platform dependent, it appears to be the same
-            # for all platforms we've tested, therefore we use `time.time` for speed.
-            #
-            # Here is the test:
-            #
-            #     $ python -c "import time;print(tuple(time.gmtime(0)[:9]))"
-            #     (1970, 1, 1, 0, 0, 0, 3, 1, 0)
-            #
-            # If you can reproduce, add to the following list of tested platforms:
-            #
-            # - Windows
-            # - macOS
-            # - Ubuntu
-            # - Alpine
-            return epoch_offset()
+def get_timestamp(dt=None):
+    """
+    Returns the number of seconds since the Unix epoch.
+    If `dt` is not specified or `None`, the current time in UTC is assumed.
+    """
+    if dt is None:
+        return epoch_offset()
```

Comment:
(nit) Might be helpful to add a comment that `epoch_offset()` will return different precision for py2 and py3.

Reply:
Done!